recipe/meta.yaml

```diff
@@ -1,17 +1,17 @@
-{% set version = "1.2.2" %}
+{% set name = "tzlocal" %}
+{% set version = "1.3" %}
 {% set hash_type = "sha256" %}
-{% set hash_val = "8e7fb9f9301039c36fb0ff0a004ec36705ac46cee99a7373694a838adc8d30b0" %}
+{% set hash_val = "ed77c6c272cecca220fce830091ec6468b677b984e1b2f1f1cfc40729728b288" %}
 
 package:
-  name: tzlocal
+  name: {{ name }}
```

Comment:
Having the name as a jinja variable isn't really needed as that, hopefully, won't change :smile: